qurl_templatetag/qurl.py

```diff
@@ -50,6 +52,15 @@ def remove(self, name, value):
         clone._qsl = [qb for qb in self._qsl if qb != (name, str(value))]
         return clone
 
+    def replace(self, name, value):
+        """ Remove value string from value parameter. """
+        clone = self._clone()
+        clone._qsl = [
+            (qb[0], qb[1].replace(value, '').strip(' +'))
```

Comment:
I think we should consider the case of percentage-encoding here.
The value could be, for instance `some thing`, `some+thing`, or `some%20thing`.
By the way, I'm really interested in the use case for this feature, could you tell me about it?

Reply:
@rlopezcc sure, I will add a more complete implementation
Use case is that you may have a search field and you want to render in the results page the search keywords as buttons where each button is linked to the same search with the selected keyword removed
something like:
`http://example.com/search/?q=some+thing` which results in `[ some x ] [ thing x ]` (where "x" is a delete button). Pardon my poor ascii art skills

Reply:
Hmm, I don't want to get in your implementation but this looks like the use case for a multi value parameter.
It could be `/?q=some&q=thing&q=another+thing`
And you could use the operators we already have: `+=, -=` to append or remove keywords.

Reply:
@rlopezcc if you have a simple text input for free text search you don't have this option: I could monkeypatch the request url in the view to use multi value parameter, but it doesn't sound the right way to accomplish this. In some cases I'm not even in control of the originating form

Reply:
Yeah, that was a possibility in my mind 🤔